### DISCUSSION TITLE
is there a list of os bootable via ipxe? ie:chromeos,grub2 bootloader? thx

### DISCUSSION THREAD

**Discussion by kocoman1** [ipxe/ipxe#931] [Q&A]
is there a list of os bootable via ipxe? ie:chromeos,grub2 bootloader? thx

**Comment by NiKiZe** [comment]
There is some collections at netboot.xyz and similar sites.

But this is a bit like asking grub which oses it supports 😉

However you can look at the iPXE banner where it says bzImage or EFI. iPXE support bzImage format in PCBIOS mode, and any efi binary in efi mode.

The question then becomes if any next stage loaders supports network boot, or has any way to be tricked into being network boot, such as using wimboot.
